console/src/main/scripts/plugins/accounts/ts/accountsGlobals.ts

```diff
@@ -60,9 +60,13 @@ module HawkularAccounts {
   }
 
   export interface IOrganizationMembership {
+    id: string;
     organization: IOrganization;
     member: IPersona;
     role: IRole;
+    $update(options:{},
+            success?:(success:ISuccessPayload) => void,
```

Comment:
ISuccessPayload and IErrorPayload might be a bit too generic. Perhaps something like IOrgMembershipSuccess or something like that...